transforms/language/lang_id/python/src/nlp.py

```diff
@@ -21,18 +21,14 @@
 
 
 def get_lang_ds_pa(table: pa.table, nlp: LangModel, col_name: str = "contents") -> tuple[pa.table, dict[str, Any]]:
-    try:
-        detected_language = pa.Table.from_pylist(
-            list(
-                map(
-                    lambda r: {"lang": r[0], "score": r[1]},
-                    map(lambda x: nlp.detect_lang(x), table[col_name].to_pylist()),
-                )
+    detected_language = pa.Table.from_pylist(
+        list(
+            map(
+                lambda r: {"lang": r[0], "score": r[1]},
+                map(lambda x: nlp.detect_lang(x), table[col_name].to_pylist()),
             )
         )
-    except Exception as e:
-        logger.warning("ERROR: %s, skipping the file", e)
```

Comment:
You should throw the exception back out through transform().  The runtime will catch it and also issue a message, but we (i.e. the runtime) wants to be told about these failures.

Reply:
Oops, thank you.

Reply:
Since I removed the code to catch exception, I suppose the updated code will throw any exception from get_lang_ds_pa() and consequently transform(). Am I missing something?